src/FetchProvider.ts

```diff
@@ -29,6 +29,7 @@ export default class FetchProvider extends BaseConnection implements Provider {
         },
       });
       const json = await response.json();
+      if (json.error) throw json.error;
```

Comment:
`error` is probably a string. Maybe we should just do `throw new ResolutionError(REC.ServiceProviderError, json.error)`